pkg/provider/kubernetes/ingress-nginx/kubernetes.go

```diff
@@ -934,6 +936,22 @@ func applyCORSConfiguration(routerName string, ingressConfig ingressConfig, rt *
 	rt.Middlewares = append(rt.Middlewares, corsMiddlewareName)
 }
 
+func applyWhitelistSourceRangeConfiguration(routerName string, ingressConfig ingressConfig, rt *dynamic.Router, conf *dynamic.Configuration) {
+	var whitelistSourceRange = ptr.Deref(ingressConfig.WhitelistSourceRange, "")
```

Comment:
nit: you can omit var here, and use := operator: `whitelistSourceRange := ptr.Deref(ingressConfig.WhitelistSourceRange, "")`